paasta_tools/api/views/instance.py

```diff
@@ -115,6 +116,45 @@ def chronos_instance_status(
     return cstatus
 
 
+def tron_instance_status(
+    instance_status: Mapping[str, Any],
+    service: str,
+    instance: str,
+    verbose: bool,
```

Comment:
Are you planning to make it less verbose later?

Reply:
No.

Reply:
We should not be showing stdout and stderr if the user does not increase the level of verbosity (with -v). It should be consistent with how paasta status looks with `paasta status -v`, `paasta status -vv`, and `paasta status -vvv` for Marathon and Chronos. 

Sorry if I was unclear. I meant that you can follow up with the changes for increasing levels of verbosity. But for the first pass, `paasta status` with no v's should return less information.

Reply:
Ah, I see. But I think this verbose variable is not included in the request from the client side. So I moved verbose to the client side.